docs/it/ciclo-del-feedback.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Il Ciclo del Feedback nello sviluppo software
+nav_order: 10
+---
+
+<!-- prettier-ignore-start -->
+# Il Ciclo del Feedback
+{: .no_toc }
+
+- TOC
+{:toc}
+
+<!-- prettier-ignore-end -->
+
+## La flessibilità del software
+
+Intuitivamente parlando, sembra ovvio che noi sviluppatori, con una connessione internet e un portatile alla mano, siamo in grado di portare a termine qualsiasi tipo di attività in tempi rapidi e con un impatto enorme sul mondo che ci circonda. Ma come è possibile?
+
+Sicuramente un fattore è la possibilità di lavorare con strumenti frutto di decenni di evoluzione tecnologica esponenziale, ma la vera ragione, il vero discriminante, sta nel prefisso "soft" nella parola "software": il prodotto del nostro lavoro è spesso intangibile e facilmente manipolabile tanto che, se commettiamo un errore, non dobbiamo prendere il semilavorato e buttarlo via, sprecando materie prime ma, nella peggiore delle ipotesi, abbiamo solo sprecato un po' di tempo.
+
+Tutto questo non è nient'altro che la "Terza rivoluzione industriale", come viene chiamata nei libri di scuola; dopo alcuni decenni in cui il collo di bottiglia è stata la potenza di calcolo, al giorno d'oggi abbiamo a disposizione CPU e memoria sufficienti per qualunque task mondano, ma questa pura forza bruta ha esposto una nuova tipologia di problemi, quelli organizzativi: far lavorare decine, se non centinaia di sviluppatori assieme sullo stesso progetto software è un problema non banale, e che tanti prima di noi si sono trovati a dover risolvere.
+
+Uno strumento che è emerso vincitore da queste difficoltà è il **ciclo del feedback**; ma di che si tratta? È un approccio in cui, a seguito di un'azione, si va a misurarne rapidamente l'effetto (il "feedback"), così da ottenerne nuove informazioni, che saranno impiegate per intraprendere i prossimi passi in maniera più conscia e informata, creando così un ciclo continuo. Questo generalmente permette di procedere in un'attività a piccoli passi ma con una discreta rapidità e, cosa ancora più importante, solitamente nella giusta direzione, visto che eventuali errori di rotta possono essere rapidamente corretti ad ogni passo.
+
+## Il manifesto Agile, l'Extreme Programming e il Devops
```

Comment:
@Jean85 mi sono appena reso conto che qui citiamo DevOps ma poi non ne parliamo. Che ne pensi di un ultimo slot di spiegazione?

Io proporrei una cosa del tipo: Dalla nascita della metodologia Agile, molte altre filosofie di sviluppo hanno preso alcuni punti del manifesto e li hanno raffinati e sviluppati nel proprio ramo di competenza. Una di queste filosofie è quella DevOps, che ha come capi saldi... I vantaggi principali sono...


Che ne dici?

Reply:
@Cadienvan aggiunto in 03a36c8, vai di review! :smile: